activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -470,9 +470,9 @@ def test_default_scope_with_references_works_with_find_by
   end
 
   test "a scope can remove the condition from the default scope" do
-    scope = DeveloperCalledJamis.david2
+    scope = DeveloperCalledJamis.jamis2
     assert_equal 1, scope.where_clause.ast.children.length
-    assert_equal Developer.where(name: "David"), scope
+    assert_equal DeveloperCalledJamis.where("salary < 10000").to_a, scope.to_a
```

Comment:
Is this change necessary?

Reply:
I have no idea, I left as the original author wrote, I will try to revert that and see what happen.